recipes/tree/build.sh

```diff
@@ -0,0 +1,2 @@
+make -j${CPU_COUNT}
```

Comment:
You can remove the patch by doing
```suggestion
make -j${CPU_COUNT} PREFIX=$PREFIX CC=$CC
```

Reply:
Thanks, fixed.

Reply:
Actually that doesn't work. Seems like the Makfile still has a hardcoded /usr/bin path.

Reply:
I reverted the changes to using the patch.

Reply:
I guess that was because of the capitalisation 
```suggestion
make -j${CPU_COUNT} prefix=$PREFIX CC=$CC
```

Reply:
I tried the lowercase version in https://github.com/conda-forge/staged-recipes/pull/11495/commits/438bceb4bb9ff02b695d3d8bd1d62641056b3231 and it failed with the same error.

Reply:
My opinion is that this builds correctly as I have it and if others want to improve the recipe they can once the feedstock is available.

Reply:
The reason we prefer to avoid patches is to make future updates easier. Rebasing patch files is tedious, even if they're trivial, and often causes updates not get merged. But there sometimes a necessary evil so that's fair enough :)

Reply:
Fair enough. I am happy to get it fixed, but I only have so many cycles and I am not proficient in compiling C code for cross platform use. Thanks for merging. I'll open issues on the repo to address the patch and osx.

Reply:
No need to open issues. If an update causes a problem you can worry about it then 😄

Thanks for adding this, I'll probably be one of the people that use it!

Reply:
Cool, glad you'll use it :) It's a nice little tool.